src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,9 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```

Comment:
UserService instantiates every time the method is invoked. 

Think about making it class level field.